pkg/controller/migmigration/quiesce.go

```diff
@@ -59,7 +115,7 @@ func (t *Task) quiesceApplications() error {
 }
 
 // Scales down DeploymentConfig on source cluster
-func (t *Task) scaleDownDeploymentConfigs(client k8sclient.Client) error {
+func (t *Task) quiesceDeploymentConfigs(client k8sclient.Client, suspend bool) error {
```

Comment:
I was confused by `suspend` here.

I get that you want to reduce duplicate code but since the majority of the code in these methods depends on whether we are quiescing or un-quiescing, it may be cleaner to just have separate methods:

`quiesceDeploymentConfigs()`
`unQuiesceDeploymentConfigs()`

etc.

Reply:
Sure.